File: src/main/java/org/jukeboxmc/item/ItemCookedSalmon.java
package org.jukeboxmc.item;

import org.jukeboxmc.item.behavior.ItemFoodBehavior;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class ItemCookedSalmon extends ItemFoodBehavior {

    public ItemCookedSalmon() {
        super ( "minecraft:cooked_salmon" );
    }

    @Override
    public float getSaturation() {
        return 9.6f;
    }

    @Override
    public int getHunger() {
        return 6;
    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/item/behavior/ItemCookedSalmon.kt
package org.jukeboxmc.server.item.behavior

import org.jukeboxmc.api.item.ItemType

class ItemCookedSalmon(itemType: ItemType, countNetworkId: Boolean) : JukeboxItemFood(itemType, countNetworkId) {

    override fun getSaturation(): Float {
        return 9.6F
    }

    override fun getHunger(): Int {
        return 6
    }
}